src/Microsoft.Extensions.Logging/ProviderAliasUtilities.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+
+namespace Microsoft.Extensions.Logging
+{
+    internal class ProviderAliasUtilities
+    {
+        private const string AliasAttibuteTypeFullName = "Microsoft.Extensions.Logging.ProviderAliasAttribute";
```

Comment:
This is by name so that 3rd party providers don't have to take an unneeded dependency ?

Reply:
Some of them cannot build against aspnetcore 2.0 (AI)